megazords/ios-rust/DEPENDENCIES.md

```diff
@@ -18,6 +18,7 @@ the details of which are reproduced below.
 * [MIT License: h2](#mit-license-h2)
 * [MIT License: http-body](#mit-license-http-body)
 * [MIT License: hyper](#mit-license-hyper)
+* [MIT License: libsqlite3-sys](#mit-license-libsqlite3-sys)
```

Comment:
I'm mildly surprised to see this appear, because `fxa-client` doesn't use sqlite.

Reply:
I'm also surprised but I simply ran the ./tools/regenerate_dependency_summaries.sh since the CI test was failing -- I might dive a little just to make sure something is not totally off.